pkg/kanx/server.go

```diff
@@ -219,7 +218,7 @@ func (s *processServiceServer) streamOutput(ss sender, p *process, fh *os.File)
 			<-t.C
 			continue
 		}
-		o := &Output{Output: buf.String()}
```

Comment:
Performs an extra allocation for the string object which may add measurable overhead considering how large the buffer is (2MB).  May want to convert this code to a 2 thread bucket brigade at some point.